app/Jobs/FinalizeSlotAttendance.php

```
@@ -0,0 +1,162 @@
+                    // even if the closed slot itself is already present/late/excused/absent)
+                    if ($existingSlots !== null
+                        && ($existingSlots['morning'] ?? null) === $mergedSlots['morning']
+                        && ($existingSlots['lunch'] ?? null) === $mergedSlots['lunch']
+                        && ($existingSlots['evening'] ?? null) === $mergedSlots['evening']
+                    ) {
+                        return 'skipped';
+                    }
+
+                    $oldDiscipline = $disciplineService->calculateDisciplineScore($user);
+
+                    AttendanceListe::updateOrCreate(
```

PHPStan (2.2.6)

[error] Call to an undefined static method App\Models\AttendanceListe::updateOrCreate().


(staticMethod.notFound)